docs/UserGuide.md

```diff
@@ -22,17 +22,19 @@ guest management tasks done faster than traditional GUI apps.
 
 1. Copy the file to the folder you want to use as the _home folder_ for your GuestBook.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the image below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
```

Comment:
I was wondering if we should change the image? Because in the entry Bernice Yu the request is "hi" which does not make much sense. What do you think?

Reply:
Agreed!